foremanctl

```diff
@@ -8,8 +8,9 @@ OBSAH_STATE=${OBSAH_BASE}/.var/lib/foremanctl
 OBSAH_PERSIST_PARAMS=true
 export OBSAH_NAME OBSAH_DATA OBSAH_INVENTORY OBSAH_STATE OBSAH_PERSIST_PARAMS
 
+ANSIBLE_LOG_PATH=/var/log/foremanctl/foremanctl.log
```

Comment:
CI fails because
```
Warning: : log file at '/var/log/foremanctl/foremanctl.log' is not writeable and we cannot create it, aborting
```

Probably should be `.var/…` by default?